docs/index.md

```diff
@@ -1,5 +1,7 @@
 # LogDNA Provider
 
+🚧 In public beta 🚧
+
 [LogDNA](https://logdna.com) is a centralized log management platform. The LogDNA Provider allows organizations to manage Views and Alerts programmatically via Terraform.
 
 ## Example Usage
```

Comment:
Ditto, I feel like we should add a sentence here saying something like:
"This example code for creating a new View and Alert is in HCL (Hashicorp Configuration Language)."